docs/get-started/with-javascript/web.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Web
+parent: JavaScript
+grand_parent: Get Started
+has_children: false
+nav_order: 1
+---
+
+# Get started with ONNX Runtime Web
+
+## Contents
+{: .no_toc }
+
```

Comment:
Need to add a link to help users jump to ORT Web tutorial section quickly.

Reply:
currently it's in section "Documentation". do you want it to be an individual section?

Reply:
It is good then.